lib/index.js

```diff
@@ -157,10 +161,16 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       // if `authenticationMethod` is `token-client-registration` then
       // register token client for it
       if(authenticationMethod === 'token-client-registration') {
-        // FIXME: get client ID from cookie
-        const clientId = 'FIXME';
+        // generate a client id
+        const clientId = await generateId({fixedLength: true});
+        // save it to the database
         await brAuthnToken.clients.set(
           {actor: null, email, account, clientId, authenticated: true});
+        // set a cookie in the response
+        const maxAge = cfg.cid.defaults.maxAge;
+        res.setHeader('Set-Cookie', `cid=${clientId}; expires=${maxAge};` +
+          `HttpOnly`);
```

Comment:
No need for backticks here.  Infact style convention prohibits backticks without template string.

Reply:
Updated.